docs/other-topics/dialect-specific-things.md

```diff
@@ -39,7 +39,8 @@ const sequelize = new Sequelize('database', 'username', 'password', {
 
 ### SQLite
 
-The underlying connector library used by Sequelize for SQLite is the [sqlite3](https://www.npmjs.com/package/sqlite3) package (version 4.0.0 or above).
+The underlying connector library used by Sequelize for SQLite is the [sqlite3](https://www.npmjs.com/package/sqlite3) npm package (version 4.0.0 or above).\
+Due to issues with deploying a new release and to fix a possible security vulnerability, it is recommended to use the @vscode/sqlite3 fork by replacing the version with `npm:@vscode/sqlite3@5.0.7`.
```

Comment:
@ephys I saw that you used fancy admonitions elsewhere, but not sure if this warrants a caution block

Reply:
I don't think an admonition is necessary here, but we may want to add a terse one about this issue [here](https://github.com/sequelize/website/blob/main/docs/getting-started.mdx), that redirects to this page

Reply:
Note that from what I've seen, docusaurus doesn't handle `\` properly. But maybe it's only in mdx files.

Reply:
It may be good to add a tiny `package.json` snippet to show how package aliases work

Reply:
I've updated it to include the snippet. What do you mean with a terse caution one?

Reply:
What I mean is that I think most people are not going to look into the "Dialect-Specific Things" page, so I think it would be interesting to add a small warning block in "Installing" on this page https://sequelize.org/v6/manual/getting-started.html :)

Reply:
Ah yes, that sounds smart. I'll add that tomorrow